Replicator/Puller.hh

```diff
@@ -93,7 +93,7 @@ namespace litecore { namespace repl {
         actor::ActorBatcher<Puller,IncomingRev> _returningRevs;
         Retained<Inserter> _inserter;
         mutable Retained<RevFinder> _revFinder;
-        Status _revFinderStatus {};
+        Status _revFinderStatus {kC4Idle};
```

Comment:
This seems to have violated the usual transition rule: worker always starts being Stopped, and transitioned to Idle after leaving Stopped. I understand your purpose. How about _revFinderStatus.level != kC4Idle || _revFinderStatus.level != kC4Stopped?

Reply:
This is not a rule, this is only a rule on certain workers.  Workers start with the rule that they are either busy or idle.

Reply:
Okay, understood.